hyperspy/io_plugins/bcf.py

```diff
@@ -1103,3 +1115,10 @@ def parse_line(line_string):
     if len(line_string) == 1:
         line_string = line_string + 'a'
     return line_string.capitalize()
+
+
+def _get_mode(obj_bcf):
+    if obj_bcf.header.sem.hv > 30.0:  # workaround to know if TEM or SEM
```

Comment:
I think that it'll be easier for the users to know what's going on if the information is written in the log. For example:

```Guessing that the acquisition instrument is TEM because the beam energy is 20 keV. If I guessed wrong, please provide the right instrument using the `instrument` keyword.```